api/src/main/resources/application.properties

```diff
@@ -7,7 +7,21 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 
 # In a Production system we would use Liquibase or a similar tool, but for the purposes of this exercise
 # we are keeping things simple and just using Hibernate DDL generator. Works just fine for our purposes.
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=validate
 spring.integration.jdbc.initialize-schema=always
 
-api.retry-after.delay=30
\ No newline at end of file
+api.retry-after.delay=30
+
+
+## -----------------------------------------------------------------------------------------------------  LOGGING LEVEL
+logging.level.root=INFO
+logging.level.gov.cms.ab2d=INFO
+
+logging.level.org.springframework=WARN
+logging.level.com.zaxxer.hikari=INFO
+logging.level.org.hibernate=WARN
+## -------------------------------------------------------------------------------------------------  LIQUIBASE LOGGING
+logging.level.liquibase.executor.jvm.JdbcExecutor=INFO
+logging.level.liquibase.integration.spring=DEBUG
```

Comment:
`DEBUG` might be too verbose

Reply:
oops... The above configuration worked best while testing. I forgot to change it back after testing. 
Will change it to `WARN` for all the liquibase shortly

Reply:
@spathiyil One more thought: I am used to the fact that _usually_ logging configuration goes into its own property file: `logging.properties`, `log4j.xml`, etc. How hard would it be to move the loggers config into a separate file so that it does not clutter `application.properties`?

Reply:
This seems to be the default way for logging in `[spring-boot](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-logging)`.

```
Spring Boot includes a number of extensions to Logback that can help with advanced configuration. .... Because the standard logback.xml configuration file is loaded too early, you cannot use extensions in it. ... The extensions cannot be used with Logback’s configuration scanning. If you attempt to do so, making changes to the configuration file results in an error similar to one of the following being logged:
```

Doing a quick read, it seems possible to get spring-boot's customization etc while creating a separate `logback-spring.xml` file. But am unsure how much effort it would be as I had not had to do it in a spring-boot project before.

Would it be okay, if we do it in a separate story outside of this PR?
For now, I could just delete the liquibase package names from here to reduce clutter, if it helps?

Reply:
@spathiyil Agreed!

Reply:
done